src/os_win32.c

```diff
@@ -7406,6 +7467,18 @@ vtp_sgr_bulks(
     vtp_printf((char *)buf);
 }
 
+    static void
+wt_init(void)
+{
+    wt_working = (mch_getenv("WT_SESSION") != NULL);
```

Comment:
Is this the only way to detect Windows Terminal?
Is there any other way like termresponse?

Reply:
The method is official.
The original URA does not appear immediately. I have a memory in Microsoft's issues.
termresponse (CSI DA) returns the same result as ConPTY.

Reply:
> termresponse (CSI DA) returns the same result as ConPTY.

Ah, Windows Terminal handles Win32 console programs via ConPTY, so it makes sense.